backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.ts

```diff
@@ -513,6 +525,68 @@ export const buildCredentialJwtPayload = (jwtData: {
   };
 };
 
+const writeVcIssuedEvent = async (
+  eventService: IEventService,
+  sessionAttributes: BiometricSessionFinishedAttributes,
+  credential: BiometricCredential,
+  audit: AuditData,
+): Promise<Result<void, void>> => {
+  const {
+    biometricSessionId,
+    govukSigninJourneyId,
+    subjectIdentifier,
+    sessionId,
+    issuer,
+    redirectUri,
+    clientId,
+  } = sessionAttributes;
+  const { credentialSubject } = credential;
+  const { contraIndicatorReasons, txmaContraIndicators, flags, flaggedRecord } =
+    audit;
+
+  const hasFlags = flags != null;
+
+  const writeEventResult = await eventService.writeGenericEvent({
+    eventName: "DCMAW_ASYNC_CRI_VC_ISSUED",
+    sub: subjectIdentifier,
+    clientId,
+    sessionId,
+    govukSigninJourneyId,
+    getNowInMilliseconds: Date.now,
+    transactionId: biometricSessionId,
+    componentId: issuer,
+    ipAddress: undefined,
+    txmaAuditEncoded: undefined,
+    redirect_uri: redirectUri,
+    suspected_fraud_signal: undefined,
+    evidence: [
+      {
+        ...credential.evidence[0],
+        ...(hasContraIndicators(credential) && {
+          ciReasons: contraIndicatorReasons,
+        }),
+        txmaContraIndicators,
+      },
+    ],
+    flaggedRecord: hasFlags
+      ? ([{ ...flaggedRecord }] as FlaggedRecord[])
```

Comment:
```suggestion
      ? flaggedRecord
```

^ any reason we can't just do this? `flaggedRecord` should already be a `FlaggedRecord[]`